src/apps/frontend/components/breadcrumb/breadcrumb-item.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+interface BreadcrumbItemProps {
+  pageName: string;
+}
+
+const BreadcrumbItem: React.FC<BreadcrumbItemProps> = ({ pageName }) => (
+  <div className="mb-6 flex flex-col gap-3 sm:flex-row sm:items-center sm:justify-between">
+    <div className="text-title-md2 font-semibold text-black dark:text-white">
+      {pageName}
+    </div>
+    <div>
+      <div className="flex items-center gap-2">
+        <div>
+          <Link className="font-medium" to="/">
+            Dashboard /
```

Comment:
Why would this belong to breadcrumb item?

I would think that that Breadcrumb takes a list of breadcrumb item (ReactNode type) and assemble them dynamically. The breadcrumb component is responsible to use `Dashboard` at the top breadcrumb item